addon/components/md-default-column-header.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+import layout from '../templates/components/md-default-column-header';
+const { computed: { alias } } = Ember;
+
+export default Ember.Component.extend({
+  layout,
+  tagName: 'th',
+  attributeBindings: ['data-field'],
```

Comment:
prefer to use something like
```js
attributeBindings: ['data-field:field'],
field: alias('column.valueBindingPath'),
```
so we avoid dasherized attribute names on the component